src/Kupo/App/Http.hs

```diff
@@ -128,6 +133,9 @@ app withDatabase patternsVar readHealth req send =
         ("v1" : "matches" : args) ->
             routeMatches (requestMethod req, args)
 
+        ("v1" : "datums" : args) ->
```

Comment:
It might be confusing that this endpoint is called "datums" but only ever returns one datum.

Reply:
This is a typical convention in rest APIs. Nouns are plural just to make things consistent.

You can always read this as "get datum with hash X from the collection of datums.

Reply:
There's nothing worse than an API which uses both singular and plural interchangeably. So one has to pick a side and, it's usually plural because more natural in most cases

Reply:
Wasn't aware of that convention but makes total sense :)